en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,102 @@
+
+Approximatively 6 weeks prior to the anticipated stable minor release, a beta minor release is published. Once a beta release is tagged, any new feature or API that didn't make it to the beta should be targeted to the follow up minor release. This allows the CMS development team to perform a regression test on the minor release. 
+
+The next minor release branch of each module is forked to a *patch release branch*. Patch release branches normally follow the pattern `MAJOR.MINOR`. For example, the patch release branch for `silverstripe/installer` 5.1 is called `5.1` and can be installed with `composer create-project silverstripe/installer:5.1.x-dev mysite`.
+
+Approximatively 2 weeks prior to the anticipated stable minor release, a release candidate is tagged. The release candidate is sent to an external auditor for a security review. The auditor normally received confidential security patches as well for review.
+
+### Stable release
+
+Once the team is ready, stable versions of all supported modules are tagged. A complete changelog is posted to the Silverstripe CMS documentation site along with any relevant upgrade notes.
+
+## Minor release support timeline
+
+Minor release support timeline follow similar phases to major release. But those phases are far more condensed.
+
+### Bug and security fixes
+
+A Silverstripe CMS minor release line enters the *bug and security fixes* phase once it is tagged *stable*.
+
+A minor release in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+It does **not** receive:
+- new feature
+- new APIs.
+
+A minor release line stays in the *bug and security fixes* phase until a follow up minor release is tagged.
+
+### Security fixes only
+
+Once the follow up minor release is tagged, the minor release that was in the *bug and security fixes* phase transition to a *security fixes only* phase.
+
+In the *security fixes only* phase, a Silverstripe CMS minor release line only receives patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+Upon entering the *security fixes only* phase, the end-of-life date for that minor release is scheduled for six months later.
+
+### End-of-life
+
+Six months after entering the *security fixes only*, the minor release goes end-of-life. No more security patches will be shipped for that minor release.
+
+### The last minor release in a major release line
+
+Shortly before tagging a new major release, one last minor release will be tagged for the major release line in active development.
+
+The last minor release in a major release line will have an extended support phases in line with our major release policy:
+- the last minor stays in the *bug and security fixes* phase for one year
+- then it enters the *security fixes only* phase for another year
+- then the minor release goes end of life along with its corresponding major release line.
```

Comment:
I'm on the fence about outlining this here, vs just changing it to this:
```suggestion
The last minor release in a major release line will have an extended support phases in line with our [major release policy](major_release_policy).
```

Reply:
Applied the suggestion in the interest of keeping the doc short and to the point.